Cargo.toml

```diff
@@ -4,4 +4,5 @@ resolver = "2"
 members = [
     "ln-resource-mgr",
     "ln-simln-jamming",
+    "reputation-builder",
```

Comment:
We're allowed to have multiple binary crates, how about this structure instead?

```
ln-simln-jamming/
├── Cargo.toml
└── src/
    ├── main.rs
    └── bin/
        ├── build_reputation.rs
```

Would allow us to de-dup some of the parsing defaults